examples/coordinates/plot_sgr-coordinate-frame.py

```diff
@@ -118,71 +118,37 @@ class Sagittarius(coord.BaseCoordinateFrame):
 D = rotation_matrix(SGR_PHI, "z", unit=u.radian)
 C = rotation_matrix(SGR_THETA, "x", unit=u.radian)
 B = rotation_matrix(SGR_PSI, "z", unit=u.radian)
-SGR_MATRIX = np.array(B.dot(C).dot(D))
+A = np.diag([1.,1.,-1.])
+SGR_MATRIX = matrix_product(A, B, C, D)
 
 ##############################################################################
-# This is done at the module level, since it will be used by both the
-# transformation from Sgr to Galactic as well as the inverse from Galactic to
-# Sgr. Now we can define our first transformation function:
+# This rotation matrix is defines at the module level, since it will be used
+# by both the transformation from Sgr to Galactic as well as the inverse from
+# Galactic to Sgr. Now we can define our first transformation function:
```

Comment:
How about:
```
# Since we already constructed our matrix above, and the inverse of a
# rotation matrix is just its, transpose, the required transformation  functions
# are very simple.
```